mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -307,6 +346,7 @@
     </CAbilResearch>
     <CAbilTrain id="FactoryTrain">
         <InfoArray index="Train25" Time="30"/>
+        <Range value="3"/>
```

Comment:
Factory's maximum spawn radius increased from 2 to 3

Other unit production structures have a spawn range of 2 (besides Larva with 4 but eggs pile up in one place).

Why is this Factory specific and not raised to 3 for all producing structures (Barracks, Starport, CC, Gateway, Hatchery, Nexus, Robotics Facility, Stargate)?